lib/tasks/import_titles.rake

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+desc 'Import CSV file to the show db'
+
+task import: :environment do
+  Show.insert_all(shows_csv)
+end
+
+private
+
+def shows_csv
+  CSV.foreach('netflix_titles.csv', headers: true).map do |row|
+    {
+      show_type: row['type'],
```

Comment:
love this, very clear

Reply:
Although since all of them are the same, you just need to modify this one maybe it would be best to do something like:
```
CSV.foreach('netflix_titles.csv', headers: true).map do |row|
    obj = row.to_h
    obj[:show_type] = obj.delete('type')
    obj
end
```
This way you convert the CSV row into hash, if you add other types you dont have to modify this thing. You just modify it for the type conversion.

Reply:
Thanks 😊  I thought, it needs refactoring, just wasn't sure how.